### PR TITLE
toss a lot of redundant mappings

### PR DESCRIPTION
**Pull request by eristhea**
a follow-up to #629

### CODE REVIEW

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 1]
```diff
@@ -195,76 +146,71 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
...
 		ARG 0 renderCall
-	METHOD renderCrosshair renderCrosshair (I)V
+	METHOD renderCrosshair (I)V
 		ARG 0 size
-	METHOD renderThreadTesselator renderThreadTessellator ()Lnet/minecraft/unmapped/C_hiiunyvh;
-	METHOD replayQueue replayQueue ()V
-	METHOD resetTextureMatrix resetTextureMatrix ()V
-	METHOD restoreProjectionMatrix restoreProjectionMatrix ()V
-	METHOD runAsFancy runAsFancy (Ljava/lang/Runnable;)V
+	METHOD runAsFancy (Ljava/lang/Runnable;)V
```
Comment (OroArmor) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping]:
```suggestion
	METHOD renderThreadTesselator renderThreadTessellator ()Lnet/minecraft/unmapped/C_hiiunyvh;
	METHOD runAsFancy (Ljava/lang/Runnable;)V
```

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 2]
```diff
@@ -1,166 +1,118 @@
 CLASS com/mojang/blaze3d/systems/RenderSystem
 	FIELD MAX_SUPPORTED_TEXTURE_SIZE maxSupportedTextureSize I
```
Comment (OroArmor) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:2]:
```suggestion
	FIELD MAX_SUPPORTED_TEXTURE_SIZE maxSupportedTextureSize I
	FIELD RENDER_THREAD_TESSELATOR RENDER_THREAD_TESSELLATOR Lnet/minecraft/unmapped/C_hiiunyvh;
```

File: mappings/net/minecraft/nbt/AbstractNbtList.mapping [hunk 3]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/unmapped/C_zveiijin net/minecraft/nbt/AbstractNbtList
 	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
-	METHOD add add (ILjava/lang/Object;)V
+	METHOD add (ILjava/lang/Object;)V
```
Comment (OroArmor) [inline mappings/net/minecraft/nbt/AbstractNbtList.mapping]:
```suggestion
	METHOD add add (ILjava/lang/Object;)V
```

File: mappings/net/minecraft/nbt/AbstractNbtList.mapping [hunk 4]
```diff
@@ -13,8 +13,8 @@ CLASS net/minecraft/unmapped/C_zveiijin net/minecraft/nbt/AbstractNbtList
 	METHOD m_wjgcgvzd addElement (ILnet/minecraft/unmapped/C_oivssbvb;)Z
 		ARG 1 index
 		ARG 2 nbt
-	METHOD remove remove (I)Ljava/lang/Object;
+	METHOD remove (I)Ljava/lang/Object;
```
Comment (OroArmor) [inline mappings/net/minecraft/nbt/AbstractNbtList.mapping]:
```suggestion
	METHOD remove remove (I)Ljava/lang/Object;
```

File: mappings/net/minecraft/nbt/AbstractNbtList.mapping [hunk 5]
```diff
@@ -13,8 +13,8 @@ CLASS net/minecraft/unmapped/C_zveiijin net/minecraft/nbt/AbstractNbtList
...
 		ARG 1 index
-	METHOD set set (ILjava/lang/Object;)Ljava/lang/Object;
+	METHOD set (ILjava/lang/Object;)Ljava/lang/Object;
```
Comment (OroArmor) [inline mappings/net/minecraft/nbt/AbstractNbtList.mapping]:
```suggestion
	METHOD set set (ILjava/lang/Object;)Ljava/lang/Object;
```
These three seem to be bridges or something:
image

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/entity/EntityRenderer.mapping [hunk 6]
```diff
@@ -14,7 +14,6 @@ CLASS net/minecraft/unmapped/C_oddtxnnw net/minecraft/client/render/entity/Entit
 		ARG 2 tickDelta
 	METHOD m_gkfhbwdx getPositionOffset (Lnet/minecraft/unmapped/C_vptppxob;)Lnet/minecraft/unmapped/C_vgpupfxx;
 	METHOD m_hknkoqwr getTexture (Lnet/minecraft/unmapped/C_vptppxob;)Lnet/minecraft/unmapped/C_ncpywfca;
-	METHOD m_idqrsxfa (Lnet/minecraft/unmapped/C_vptppxob;)F
```
Comment (OroArmor) [inline mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: This is something to look into for the javadocs:
image

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:1]: Shouldn't all of these fields be be `SCREAMING_SNAKE`?

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping [hunk 7]
```diff
@@ -1,41 +1,36 @@
...
+	METHOD readResource (Ljava/io/InputStream;)Ljava/nio/ByteBuffer;
 		ARG 0 inputStream
 	METHOD readResource (Ljava/nio/channels/ReadableByteChannel;I)Ljava/nio/ByteBuffer;
 		ARG 0 bytes
 		ARG 1 defaultSize
-	METHOD releaseTextureId releaseTextureId (I)V
+	METHOD releaseTextureId (I)V
 		ARG 0 id
-	METHOD writeAsPNG writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIII)V
+	METHOD writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIII)V
```
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/texture/TextureUtil.mapping]:
```suggestion
	METHOD writeAsPNG writeAsPng (Ljava/nio/file/Path;Ljava/lang/String;IIII)V
```

File: mappings/net/minecraft/network/PacketByteBuf.mapping
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:1]: All the `LE`/`BE` in this, `BufferHelper`, and `RconClient` should be `Le`/`Be`.
They should also probably have javadocs saying they stand for little/big endian (I assume).

Not sure if either of those belong in this PR though.

File: mappings/net/minecraft/util/DelegatingDataOutput.mapping [hunk 8]
```diff
@@ -1,32 +1,32 @@
...
+	METHOD writeInt (I)V
 		ARG 1 i
-	METHOD writeLong writeLong (J)V
+	METHOD writeLong (J)V
 		ARG 1 l
-	METHOD writeShort writeShort (I)V
+	METHOD writeShort (I)V
 		ARG 1 i
-	METHOD writeUTF writeUTF (Ljava/lang/String;)V
+	METHOD writeUTF (Ljava/lang/String;)V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/util/DelegatingDataOutput.mapping]:
```suggestion
	METHOD writeUTF writeUtf (Ljava/lang/String;)V
```

File: mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping [hunk 9]
```diff
@@ -57,7 +57,7 @@ CLASS net/minecraft/unmapped/C_rcqaryar net/minecraft/entity/ai/brain/task/TaskB
 			ARG 3 function2
 			ARG 4 function3
 			ARG 5 function4
-		METHOD lift1 lift1 (Lcom/mojang/datafixers/kinds/App;)Ljava/util/function/Function;
+		METHOD lift1 (Lcom/mojang/datafixers/kinds/App;)Ljava/util/function/Function;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping:60]: What is this, and why the `1`?
Enigma couldn't find any calls, is it just dead code?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping
Comment (eristhea) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:1]: only the final ones, i'm pretty sure everything here follows convention

File: mappings/net/minecraft/network/PacketByteBuf.mapping
Comment (eristhea) [reply mappings/net/minecraft/network/PacketByteBuf.mapping:1]: I'll pull that into an issue for another PR

**Comment by eristhea** [comment]
adding suggestions individually because the file tab is huge. sorry

File: mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping (hunk 9 above)
Comment (eristhea) [reply mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping:60]: going to leave this for another PR when i cover entities, i've set up a reminder on discord

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping
Comment (supersaiyansubtlety) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:1]: mb, didn't notice so few were final
looks like there are a few though, I'll suggest them individually

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 10]
```diff
@@ -1,166 +1,119 @@
...
-	FIELD MINIMUM_ATLAS_TEXTURE_SIZE MINIMUM_ATLAS_TEXTURE_SIZE I
 	FIELD RENDER_THREAD_TESSELATOR RENDER_THREAD_TESSELLATOR Lnet/minecraft/unmapped/C_hiiunyvh;
-	FIELD apiDescription apiDescription Ljava/lang/String;
-	FIELD isInInit isInInit Z
-	FIELD lastDrawTime lastDrawTime D
 	FIELD modelViewStack MODEL_VIEW_STACK Lorg/joml/Matrix4fStack;
-	FIELD pollEventsWaitStart pollEventsWaitStart Ljava/util/concurrent/atomic/AtomicLong;
-	FIELD pollingEvents pollingEvents Ljava/util/concurrent/atomic/AtomicBoolean;
```
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:10]: can't suggest on deleted lines, but these are `static final`

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping (hunk 10 above)
Comment (eristhea) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:10]: the uppercasing doesn't apply to atomics (i forgot too, thanks linter)

Comment (supersaiyansubtlety) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:10]: oh, didn't know, whoops

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 11]
```diff
@@ -196,72 +147,66 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
...
-	METHOD setShaderColor setShaderColor (FFFF)V
+	METHOD setShaderColor (FFFF)V
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
-	METHOD setShaderFog setShaderFog (Lnet/minecraft/unmapped/C_vidgpspy;)V
-		ARG 0 fog
-	METHOD setShaderGameTime setShaderGameTime (JF)V
+	METHOD setShaderGameTime (JF)V
```
Comment (OroArmor) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:165]:
```suggestion
	METHOD setShaderFog (Lnet/minecraft/unmapped/C_vidgpspy;)V
		ARG 0 fog
	METHOD setShaderGameTime (JF)V
```

**Review by OroArmor — CHANGES_REQUESTED** [review]

**Review by OroArmor — APPROVED** [review]

**Pull request merged**